lib/ivis_opengl/gfx_api.h

```diff
@@ -747,6 +753,13 @@ namespace gfx_api
 		float fogBegin;
 		float timeState; // graphicsCycle
 		int fogEnabled;
+		int viewportWidth;
+		int viewportheight;
+		float unused2;
```

Comment:
I don't really know how to elegantly handle std140 layout, I added an extra padding.